qiskit/synthesis/__init__.py

```diff
@@ -89,6 +89,28 @@
 
 .. autofunction:: synth_qft_line
 
+Unitary Synthesis
+=================
+
+.. autofunction:: qs_decomposition
+
+One-qubit Synthesis
+===================
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   OneQubitEulerDecomposer
+
+Two-qubits Synthesis
```

Comment:
For consistency, I think we should use `Two-Qubit` (without the final `s`) to match the class name 🙂 The same holds for the import name, because right now we have imports like `....two_qubits.two_qubit_decompose`